File: Python/1768. Merge Strings Alternately.py
# 1768. Merge Strings Alternately
# Level: Easy
# https://leetcode.com/problems/merge-strings-alternately/

class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        str = ""
        i = 0;
        while i < len(word1) or i < len(word2):
            if i < len(word1):
                str += word1[i]
            if i < len(word2):
                str += word2[i]
            i += 1
        
        return str

File: TypeScript/1768. Merge Strings Alternately.ts
// 1768. Merge Strings Alternately
// Level: Easy
// https://leetcode.com/problems/merge-strings-alternately/

function mergeAlternately(word1: string, word2: string): string {
    let str = "";
    for(let i = 0; i < word1.length || i < word2.length; i++){
        if(i < word1.length)
            str += word1[i];
        if(i < word2.length)
            str += word2[i];
    }
    return str;
};